src/Simulation/EntryPointDriver/Azure.cs

```diff
@@ -235,9 +235,16 @@ internal sealed class AzureSettings
 
         /// <summary>
         /// The base URI of the Azure Quantum endpoint.
+        /// If both BaseUri and Location properties are not null, BaseUri takes precedence.
```

Comment:
The tags can help make sure the doc comment stays up to date if the property names change in the future:

```suggestion
        /// If both <see cref="BaseUri"/> and <see cref="Location"/> properties are not null, <see cref="BaseUri"/> takes precedence.
```